client/comms/wsconn.go

```diff
@@ -393,10 +403,6 @@ func (conn *wsConn) Send(msg *msgjson.Message) error {
 // Request sends the message with Send, but keeps a record of the callback
 // function to run when a response is received.
 func (conn *wsConn) Request(msg *msgjson.Message, f func(*msgjson.Message)) error {
-	if !conn.isConnected() {
-		return fmt.Errorf("cannot send on a broken connection")
-	}
-
```

Comment:
Because this check happens at the beginning of Send, I thought it was redundant to have it here too.